src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final int index = new Random().nextInt(Color.values().length);

```

Comment:
Did you check how it works? Can you get a random color by calling the `getRandomColor()` method?